opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -416,7 +417,7 @@ export const roleEditContext = async (context, user, roleId, input) => {
 };
 
 export const assignOrganizationToUser = async (context, user, userId, organizationId) => {
-  if (!isUserHasCapability(user, SETTINGS_SET_ACCESSES) && isUserHasCapability(user, VIRTUAL_ORGANIZATION_ADMIN)) {
+  if (!isAdminWithoutSettingsSetAccessCapability(user)) {
```

Comment:
should be `if (isAdminWithoutSettingsSetAccessCapability(user))`, without the `!`